posts/compliance-users-v3.md

```diff
@@ -0,0 +1,15 @@
+---
+version: 1.0.0
```

Comment:
assuming this version is different that the v3 elsewhere, but wanted to flag in case.

Reply:
ah good flag! this `version` is something we don't use too much, but reflects the version of the _post itself_ (rather than the v3 which we're using to describe the compliance users tool). if we make an update to the post after it's launched, best practice is to increment the version here.